### DISCUSSION TITLE
Output to Word Document

### DISCUSSION THREAD

**Discussion by tbeamer-s4nets** [cammurray/orca#311] [Q&A]
Has anybody put something together to get the ORCA Report to output to a Word document?  THanks!

**Comment by cammurray** [comment]
@tbeamer-s4nets not sure i've seen people do this, but it's technically possible. the way ORCA is architected is the outputs are modular, that's how we support output to CSV, CosmosDB, HTML (default), etc.. someone just needs to write the module for outputting the content to the format you want.

be interested to hear your use case. I am assuming for some kind of reporting?

**Comment by tbeamer-s4nets** [comment]
I use the ORCA tool as one component for doing M365 assessments for customers.  Being able to export direct to Word would make it much easier to preserve the formatting of the HTML output when putting the contents into a much more comprehensive word document.

Sent via Outlook<https://aka.ms/qtex0l> for iOS, powered by Office 365
Tim Beamer
Vice President, Microsoft Solutions
***@***.***
412-944-9701
________________________________
